src/Components/ErrorBoundary/index.js

```diff
@@ -0,0 +1 @@
+export { ErrorBoundary } from './ErrorBoundary';
```

Comment:
I don't see purpose behind this file. 

Also, one important thing that React Components should have 'jsx' extension